projects/plugins/jetpack/modules/widget-visibility/widget-conditions.php

```diff
@@ -63,6 +64,7 @@ public static function init() {
 			// On a screen that is hosting the API in the gutenberg editing experience.
 			if ( is_customize_preview() || 'widgets.php' === $pagenow ) {
 				$add_data_assets_to_page = true;
+				$add_block_controls      = true;
```

Comment:
Would it make sense to add the settings to the regular block editor too in the future? I feel it could be useful there, although I can see how confusing it would be to have a feature called widget visibility impact and be useful when editing a post or a page.

Reply:
I agree this is a really powerful feature that could be opened up across all contexts of the site; if we go that way I'd like to handle it in a second step with more planning and discussion.

Reply:
We are currently not, and I'm not sure how to add this.